pinery-lims/src/main/java/ca/on/oicr/pinery/lims/SampleAttribute.java

```diff
@@ -29,7 +29,7 @@
     SAMPLE_RECEIVE_DATE("geo_receive_date", "Receive Date"),
     EXTERNAL_NAME("geo_external_name", "External Name"),
     TUBE_ID("geo_tube_id", "Tube ID"),
-    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing"),
+    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing", "Targeted Sequencing"),
```

Comment:
"Targeted Sequencing" is MISO's attribute name (see https://github.com/oicr-gsi/pinery/pull/19/commits/658e31d9b77e9eb88e5d50ade166460133349559), where as "Targeted Resequencing" is GSLE's attribute name.  "Targeted Sequencing" is the more correct name, correct @callunity?  This enum's N:1 mapping functionality is to standardize different lims' attribute names.